packages/firebase/index.d.ts

```diff
@@ -3119,6 +3119,14 @@ declare namespace firebase.auth {
      * Sets the current language to the default device/browser preference.
      */
     useDeviceLanguage(): void;
+    /**
+     * Modify this Auth instance to communicate with the Firebase Auth emulator.  This must be 
+     * called synchronously immediately following the first call to `firebase.auth()`.  Do not use
+     * with production credentials as emulator traffic is not encrypted.
+     * 
+     * @param url - The URL at which the emulator is running (eg, 'http://localhost:9099')
```

Comment:
```suggestion
     * @param url The URL at which the emulator is running (eg, 'http://localhost:9099')
```

We are using JSDoc currently, which doesn't requires the `-` after the  parameter name